cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/search/SearchWriterSuite.scala

```diff
@@ -387,4 +430,135 @@ class SearchWriterSuite extends TestBase with AzureSearchKey with IndexLister
     retryWithBackoff(assertSize(in, 2))
   }
 
+
+  test("no vector,  index json, new index") {
+    //create new index and add docs
+    lazy val in1 = generateIndexName()
```

Comment:
you dont technically need the lazys when you are in the test scope, (Thanks for noticing though and trying to fit in! most people dont!)

These lazy markers help us perform reflection efficiently during code and test generation (separate part of the pipeline) but things inside the test() {...} are exempt from needing the lazy thing because those test functions arent called in our codegen system (but we do need to instantiate the classes hence why we have lots of lazys on fields that would run if the test class was instantiated)